### PR TITLE
EZP 30001 : Limiting "Content\Create" to Section disables "Create" button in COTF

### PR DESCRIPTION
**Pull request by pawbuj**
| Question      | Answer
| ------------- | ---
| Tickets       | [EZP-30001](https://jira.ez.no/browse/EZP-30001)
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Tests pass?   | yes
| Doc needed?   | no
| License       | [GPL-2.0](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/LICENSE)

#### Checklist:
- [x] Coding standards (`$ composer fix-cs`)
- [x] Ready for Code Review

### CODE REVIEW

File: src/bundle/Controller/ContentOnTheFlyController.php [hunk 1]
```diff
@@ -159,4 +160,19 @@ public function hasCreateAccessAction(
 
         return $response;
     }
+
+    /**
+     * @param Location $location
```
Comment (adamwojs) [inline src/bundle/Controller/ContentOnTheFlyController.php]: Please use FQCN inside docblocks in this repository

File: src/bundle/Controller/ContentOnTheFlyController.php [hunk 2]
```diff
@@ -159,4 +160,19 @@ public function hasCreateAccessAction(
...
+     * @param ContentType $contentType
+     * @param string $language
+     *
+     * @return ContentCreateStruct
+     */
+    private function createContentCreateStruct(Location $location, ContentType $contentType, string $language): ContentCreateStruct
```
Comment (webhdx) [inline src/bundle/Controller/ContentOnTheFlyController.php]: Nitpick: Break into multiple lines as this is too lengthy.

**Comment by micszo** [comment]
@lserwatka ready

**Comment by mnocon** [comment]
@lserwatka @micszo the test failure is unrelated to this PR, this is a problem with DatePicker in tests (I'm trying to solve it in https://github.com/ezsystems/ezplatform-admin-ui/pull/797/files)

**Pull request merged**